contrib/devtools/release_management/release_management_utils.py

```diff
@@ -179,10 +179,17 @@ def insert_line_into_file(file_path: str, line_number: int, line_content: str):
     with open(file_path, "w") as file:
         file.writelines(lines)
 
-def ask_for_step_skip(script_step: str, config: dict):
-    config[k_script_steps][script_step] = {}
-    config[k_script_steps][script_step][k_stop] = False
-    if (input("Do you want to skip this step? (Y/N)").upper() == "Y"):
+def ask_for_step_skip(script_step: str, config: dict, step_details: str = ""):
```

Comment:
The usage of the `config` variable is a bit confusing: it is declared and initialized as global in the other two scripts, hence explicitly passing that shouldn't be mandatory. 
Before this change, this function was called without explicitly passing the dict.

I would suggest to declare and define this variable in the global space of this file, and import that in the two other scripts. This would be enough for removing `config` from this function signature + each invocation, without affecting the visibility on the other scripts.

Reply:
I saw the new change, looks good to me